src/SignalR/server/Core/src/HubConnectionContext.cs

```diff
@@ -48,37 +48,17 @@ public class HubConnectionContext
         /// Initializes a new instance of the <see cref="HubConnectionContext"/> class.
         /// </summary>
         /// <param name="connectionContext">The underlying <see cref="ConnectionContext"/>.</param>
-        /// <param name="keepAliveInterval">The keep alive interval. If no messages are sent by the server in this interval, a Ping message will be sent.</param>
         /// <param name="loggerFactory">The logger factory.</param>
-        public HubConnectionContext(ConnectionContext connectionContext, TimeSpan keepAliveInterval, ILoggerFactory loggerFactory)
```

Comment:
Question: Should we keep the 2 old constructors so this isn't a breaking change?

Reply:
Meh.

Reply:
@anurse Do we need to make an announcement?

Reply:
Sure do!